spec/support/features/trainee_steps.rb

```diff
@@ -29,5 +29,13 @@ def summary_page
     def confirm_details_page
       @confirm_page ||= PageObjects::Trainees::ConfirmDetails.new
     end
+
+    def and_i_confirm_my_details(checked: true, section:)
+      checked_option = checked ? "check" : "uncheck"
+      confirm_details_page ||= PageObjects::Trainees::ConfirmDetails.new
```

Comment:
`confirm_details_page` is already defined above. If you remove it the tests still pass.

Reply:
good spot https://github.com/DFE-Digital/register-trainee-teachers/pull/176/commits/7e11f2317b5e083113e8eca1b35f16704c43452b